docs/uninstall/uninstall_everest_helm.md

```diff
@@ -0,0 +1,232 @@
+# Uninstall Percona Everest using Helm
+
+If you installed Everest using Helm, follow the instructions below to uninstall.
+
+
+## Uninstall the database namespaces
+
+If you have created any database namespaces other than the default namespace, ensure to delete them first.
+
+
+```sh
+helm uninstall everest -n <your_db_namespace>
+kubectl delete ns <your_db_namespace>
+```
+
+!!! note
+    This runs a chart hook that cleans up your database resources. Although it is not recommended, you can skip this step by specifying `cleanupOnUninstall=false`.
+
+
+## Uninstall Percona Everest
+
+To uninstall Percona Everest, follow these steps:
+{.power-number}
+
+1. Run the following command:
+
+    ```sh
+    helm uninstall everest-core -n everest-system
+    kubectl delete ns everest-system
+    ```
+
+2. Remove the unused [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+    !!! caution alert alert-warning "warning"
+
+        Deleting CRDs can potentially cause issues with any custom resources that depend on those definitions within the kubernetes cluster. Ensure that you do not remove any CRDs that are being used by operator deployments outside of Everest.
+
+## Remove CRDs
```

Comment:
This section is out of date, please find the up-to-date list of CRDs in the main branch.

Reply:
Fixed.